### PR TITLE
Node version updates

### PR DESCRIPTION
**Pull request by DavidReque**
hi, check the changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,7 +76,7 @@
     "pino": "^9.3.2",
     "pino-pretty": "^11.0.0",
     "pluralize": "^8.0.0",
-    "rimraf": "^5.0.5",
+    "rimraf": "^^5.0.5",
```
Comment (ozziest) [inline package.json:79]:
```suggestion
    "rimraf": "^5.0.5",
```

**Review by ozziest — CHANGES_REQUESTED** [review]
Hey! I just requested one small change. ☺️

**Comment by DavidReque** [comment]
@ozziest I have made the requested changes.

**Review by ozziest — APPROVED** [review]
Looks great! 👏

**Comment by ozziest** [comment]
@DavidReque , we have an error on tests;

> npm error `npm ci` can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.

You need to execute `npm install` once.

**Comment by DavidReque** [comment]
hey @ozziest I have run `npm install` to update the file

**Comment by ozziest** [comment]
Hey @DavidReque !
We are very close to merging this PR. I've noticed some issues on the integration tests and fixed them all. But you need to rebase your PR one more time. ☺️

**Comment by ozziest** [comment]
Hey @DavidReque . The PR has some conflict. Could you rebase it with `git pull --rebase origin develop`? one more time? ☺️

**Comment by ozziest** [comment]
Hey @DavidReque, the PR still has some conflicts. Could you rebase it, please? ☺️

**Pull request closed without merging**